modules/core/src/main/scala/org/apache/spark/sql/ammonitesparkinternals/AmmoniteSparkSessionBuilder.scala

```diff
@@ -303,7 +303,7 @@ class AmmoniteSparkSessionBuilder
     )
     classServerOpt = Some(classServer)
 
-    config("spark.repl.class.uri", classServer.uri.toString)
+    config("spark.repl.class.uri", s"${classServer.uri.toString}/")
```

Comment:
@piiteraq you do have a sparkVersion here if you need to make a pivot temporarily to see if that fixes the tests.

https://github.com/alexarchambault/ammonite-spark/blob/9fb5e6a6f56412e7db68059e11666c5878f15bbe/modules/core/src/main/scala/org/apache/spark/sql/ammonitesparkinternals/AmmoniteSparkSessionBuilder.scala#L31